backend/docker-compose.test.yaml

```
@@ -21,6 +33,8 @@ services:
       context: ../
       dockerfile: Dockerfile
       network: host
+    depends_on:
+      - postgres
     ports:
       - "8002:8000"
     depends_on:
```

There's already a depends_on postgres... or am I missing something?

---

Hmmm, interesting. We've seen this fail in at least one scenario resulting in the whole "depends" being silently ignored.
I've added a comment to clarify.

---

(It broke in @pothos 's set-up, maybe Kai could tune in...)

---

Could it be related to the `service_health` condition, not sure… how about solving this race with an automatic restart of the nebraska container?

---

The port is not reachable in my case and the startup is <1s delayed, maybe the health check doesn't say whether the port is reachable?

---

Seems podman-compose [is the problem](https://github.com/containers/podman-compose/pull/453) because it started the container while the other one was still in state `(starting)`, I could try docker-compose as root on top of the Podman socket or we add `restart: always` which mitigates the problem for me.